p2p/src/pubsub.rs

```diff
@@ -164,9 +168,9 @@ where
                                 })
                                 .await?
                             {
-                                Ok(_) => net::ValidationResult::Accept,
+                                Ok(_) => net::types::ValidationResult::Accept,
                                 Err(ProcessBlockError(BlockError::BlockAlreadyExists(_id))) =>
-                                    net::ValidationResult::Accept, // TODO: ignore?
+                                    net::types::ValidationResult::Accept, // TODO: ignore?
                                 Err(err) => {
                                     // TODO: report misbehaviour to swarm manager and close connection
                                     log::error!(
```

Comment:
There's an overuse of `:?` all over the place. Please keep in mind that this makes logs much bigger and unreadable. Perhaps adding the Display trait to many things may be a good solution. Something like the PeerId, which is just a number, doesn't need to be printed as Debug trait. Maybe also MessageId? Even the errors too! You can postpone this to be done later. But we don't want our logs to be flooded with code symbols as much as possible.

Reply:
I'll convert them to use `Display` as this maintenance journey takes me through P2P. I want to remind though that almost none of the errors of P2P had implement `Display` until two days ago so it's not that surprising they are printed using `{:?}`.